partitionmanager/bootstrap.py

```diff
@@ -73,7 +89,7 @@ def _plan_partitions_for_time_offsets(
     for (i, offset), is_final in partitionmanager.tools.iter_show_end(
         enumerate(time_offsets)
     ):
-        increase = [x * offset / RATE_UNIT for x in rate_of_change]
+        increase = [x * (offset / RATE_UNIT) for x in rate_of_change]
```

Comment:
How do the parentheses make a difference? Something to do with rounding behavior?

Reply:
It's actually about integer overflow in the Python interpreter. We need to get from milliseconds down to RATE_UNIT before multiplying.

Reply:
Oh, neat. That's maybe worth a comment so that nobody deletes these load-bearing parentheses in the future. Your call.